execution/gethexec/blockchain.go

```diff
@@ -37,7 +37,7 @@ type CachingConfig struct {
 	SnapshotRestoreGasLimit            uint64        `koanf:"snapshot-restore-gas-limit"`
 	MaxNumberOfBlocksToSkipStateSaving uint32        `koanf:"max-number-of-blocks-to-skip-state-saving"`
 	MaxAmountOfGasToSkipStateSaving    uint64        `koanf:"max-amount-of-gas-to-skip-state-saving"`
-	StylusLRUCache                     uint32        `koanf:"stylus-lru-cache"`
+	StylusLRUCacheCapacity             uint32        `koanf:"stylus-lru-cache-capacity"`
```

Comment:
Can we call this `StylusLRUCacheCapacitMB` or just `StylusLRUCacheMB` to let someone reading the code see right away the unit of the `uint32`?

Reply:
I see. Okay. Let's leave it without then. 😢